### DISCUSSION TITLE
Get area to sign from PDF

### DISCUSSION THREAD

**Discussion by Max13** [creagia/laravel-sign-pad#6] [Q&A]
Hi there,

Looks promising! Sorry to open an issue but the discussion doesn't seem to be opened yet.

Let's say I have a currently existant PDF or docx, with a defined area to sign. What is the quickest and easiest way to know the area coordinates and size in px to feed your package please?

**Comment by dtorras** [comment]
Hello @Max13! no problem, I've opened the discussions tab and migrated the issue.

You should calculate the x and y position of the signature position from the top left corner of the selected page in pixels (at 72 dpi according to TCPDF docs https://tcpdf.org/docs/srcdoc/TCPDF/classes-TCPDF/#method_Image).

**Reply by Max13** [reply]
Hey! I know the theory about it, what I was looking for is actually an "easy" way to find the area coordinate from an existing PDF.

It would be complicated for a lambda user to try an area and adjust multiple time, is there a quick way on windows and macOS to (for instance) draw a rectangle on a PDF and get its coordinate in px ?

**Reply by dtorras** [reply]
I understand... I think there is no 'easy' way. In my case, I have always ended up adjusting the measurements with trial and error.

**Reply by Max13** [reply]
I would be great to be able to upload a PDF, then to select overlay an area to retrieve and store it in database for instance. It will allow end users to create signable PDF without a tech

**Reply by dtorras** [reply]
That would be a nice experience for users in some cases, but I think this would be out of this package's scope because it could be too different in every project.
